lua/hfcc/hf.lua

```diff
@@ -59,6 +59,11 @@ M.fetch_suggestion = function(request, callback)
   if api_token == "" then
     vim.notify("[HFcc] api token is empty, suggestion might not work", vim.log.levels.WARN)
   end
+  local curl_options = ""
+  local tls_skip_verify_insecure = config.get().tls_skip_verify_insecure
+  if tls_skip_verify_insecure == true then
+    curl_options = curl_options .. " --insecure "
+  end
```

Comment:
How about creating a `local function create_curl_options()` that would be called in place of `curl_options` below?